docs/blog/posts/2025-08-14-project-status-markers.md

```diff
@@ -0,0 +1,228 @@
+---
+title: PyPI now serves project status markers in API responses
+description: >
+  PyPI has implemented PEP 792, and is now serving
+  project status markers in its standard HTML and JSON APIs.
+authors:
+  - woodruffw
+date: 2025-08-14
+tags:
+  - security
+  - integrations
+---
+
+PyPI now serves [project status markers] in its standard
+[index APIs]. This allows downstream consumers (like Python package installers and
+index mirrors) to retrieve project statuses programmatically and use them to
+inform users when a project is archived or quarantined.
+
+## Summary
+
+* PyPI has implemented project status markers as proposed and accepted in
+  [PEP 792].
+* As of today, PyPI supports three standard statuses: **active** (the default),
+  **archived**, and **quarantined**.
+* Downstream consumers can now retrieve these statuses via the standard index APIs
+  and use them to inform users about the state of a project.
+
+See the [project archival] and [project quarantine] announcement posts
+for additional information on PyPI's implementation of those individual statuses.
+
+<!-- more -->
+
+## Background
+
+Many Python regularly find themselves asking the same
+questions again and again when evaluating a new dependency:
+
+* Is the dependency deprecated, potentially in favor of another project?
+* If a vulnerability is discovered in the dependency, is it likely to be
+  patched?
+* Can I expect major future changes to the dependency, or is it
+  "done" (i.e. feature complete)?
+
+These questions (and many others in the domain of supply chain security)
+essentially boil down to a single question:
+**what is the status of this project?**
+
+## The status quo before status markers
+
+Before [PEP 792], Python packaging had no less than three overlapping
+solutions for determining a project's status:
```

Comment:
In re-reading this, I now realize this is kind of stating "and now we've added a 4th" without actually saying that.